db/migrations/00001_create_nodes_table.sql

```diff
@@ -6,7 +6,7 @@ CREATE TABLE eth_nodes
     genesis_block VARCHAR(66),
     network_id    NUMERIC,
     eth_node_id   VARCHAR(128),
-    UNIQUE (genesis_block, network_id, eth_node_id)
+    UNIQUE (genesis_block, network_id, eth_node_id, client_name)
```

Comment:
Should have mentioned this on the other PR but heads up that we need to avoid modifying earlier migrations starting asap if we're working against a deployed db - otherwise the changes won't take hold.

Reply:
🤦‍♂ oh boy, that's gonna take some getting used to haha